### PR TITLE
feat(directus-next): adding draft route code

### PR DESCRIPTION
**Pull request by yanmorinokamca**
## Issue Link
https://okamca.atlassian.net/browse/STACK-148

## Implementation details
- [ ] Import code from vanier for GET /api/draft
- [ ] Adding version argument in route
- [ ] Allow callback to redefine error response and secret fetching.

## PR Checklist      
- [ ] Lint must pass
- [ ] Build must pass
- [ ] There should be no warnings/errors in the console/terminal (check locally)

## How to test this PR
- TODO

## TODO
- Refactor vanier?
- Implement in cepsum ?

## Already done
- Add external import to next and radash
- Create package <- will use system of release

### CODE REVIEW

**Review by JeromeTrottier — COMMENTED** [review]
There should probably be a demo available in the nextjs app setup with Agency OS

**Review by JeromeTrottier — COMMENTED** [review]
JSON is unable to parse language params because of automatic url formatting 
image

**Comment by yanmorinokamca** [comment]
> JSON is unable to parse language params because of automatic url formatting 

Can't get this error in chrome or firefox.

Add a try / catch on your side, according to your invalid url.

File: apps/demo/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Visit http://localhost:4200
+
+## Testing
+
+### Testing draft api
+You can test the redirect with this url:
+
+```
+http://localhost:4200/api/draft?urls=[%22%2Fdirectus-block%22]&languages=[%22en-CA%22]&secret=123
```
Comment (poclerson) [inline apps/demo/README.md:29]: J'ai remplacé par `http://localhost:4200/api/draft?urls=%5B%22%2Fdirectus-block%22%5D&languages=%5B%22en-CA%22%5D&secret=123` et ça fonctionne, j'imagine c'est les brackets qui brisent le parse du json

**Review by poclerson — APPROVED** [review]

**Pull request merged**